src/mpid/common/thread/mpidu_thread_fallback.h

```diff
@@ -666,12 +788,18 @@ do {                                                                    \
 @*/
 #define MPIDU_Thread_cond_wait(cond_ptr_, mutex_ptr_, err_ptr_)         \
     do {                                                                \
+        int saved_count = (mutex_ptr_)->count;                          \
+        MPL_thread_id_t saved_owner = (mutex_ptr_)->owner;              \
```

Comment:
The current code is already inconsistent because we don't have a coding standard that forces this naming convention. I never understood the rational. If we fall into a *shadow variable* case, the compiler would tell us and we can choose to leave it or fix it.

Reply:
I will use the the trailing underscore just for consistency within the same macro